desktop/src/main/java/bisq/desktop/util/validation/PhoneNumberValidator.java

```diff
@@ -161,4 +173,14 @@ private ValidationResult validateIncludedCountryDialingCode(String rawInput,
             return new ValidationResult(false, Res.get("validation.invalidInput", t.getMessage()));
         }
     }
+
+    private ValidationResult validateRequiredLength(String normalizedPhoneNumber, String normalizedCallingCode) {
+        try {
+            return ((0 == requiredLength) || ((normalizedPhoneNumber.length() - normalizedCallingCode.length() - 1) == requiredLength))
+                    ? new ValidationResult(true)
+                    : new ValidationResult(false, Res.get("validation.phone.incorrectLength", requiredLength));
```

Comment:
This is a new resource string currently missing from this pull request.  Do not merge until it is defined in every language!